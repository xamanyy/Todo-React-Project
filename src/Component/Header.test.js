import React from "react";
import Basic from "../Component/Header.js";

import Enzyme, { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

it("renders correctly enzyme", () => {
  const wrapper = shallow(<Basic />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
