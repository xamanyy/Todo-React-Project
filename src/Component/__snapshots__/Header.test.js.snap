// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<div>
  <nav>
    <h1>
      TODO
    </h1>
    <ul
      className="nav-items"
    >
      <li>
        <a
          className="nav-cta"
          href="#"
        >
          Home
        </a>
      </li>
      <li>
        <a
          className="nav-cta"
          href="#"
        >
          About Us
        </a>
      </li>
      <li>
        <a
          className="nav-cta"
          href="#"
        >
          Contact Us
        </a>
      </li>
    </ul>
    <div
      className="search-box"
    >
      <img
        alt="none"
        color="#777"
        height="24px"
        padding-left="20px"
        src="search.svg"
      />
      <input
        className="inpt"
        placeholder="Anything you want to search"
        type="search"
      />
    </div>
  </nav>
</div>
`;
